IQSS/dataverse-frontend
# 265 - Add breadcrumbs

## What this PR does / why we need it:
This PR adds the breadcrumbs to the File Page, Dataset Page and Home. This PR includes only the UI and the data model, the breadcrumbs are not connected yet to the different pages because we need some data from the API

## Which issue(s) this PR closes:

- Closes #265 

## Special notes for your reviewer:
Only the UI, not functional yet

## Suggestions on how to test this:
1. Install the necessary dependencies using `npm install`.
5. Build the design system with `cd packages/design-system && npm run build`.
6. Navigate back to the root directory with `cd ../../`.
7. Start Storybook with `npm run storybook`.
8. Visit [the Storybook](http://localhost:6006) and check that the File Page, the Dataset Page and the Home have the corresponding breadcrumbs at the top of the page.

## Does this PR introduce a user interface change? If mockups are available, please link/include them here:
![Captura de pantalla 2023-12-14 a las 12 14 24](https://github.com/IQSS/dataverse-frontend/assets/23359572/e85ae9a2-68ad-4a64-b86d-1c4be71722c9)

## Is there a release notes update needed for this change?:

## Additional documentation:
